shopinvader/services/customer.py

```diff
@@ -62,6 +83,31 @@ def _validator_create(self):
         )
         return schema
 
+    def _validator_update(self):
+        address = self.component(usage="addresses")
+        schema = address._validator_update()
+        exclude_keys = ["type"]
+        for key in exclude_keys:
+            schema.pop(key, None)
+        schema.update(
+            {
+                "email": {"type": "string", "required": False},
+                "external_id": {"type": "string", "required": False},
+                "vat": {"type": "string", "required": False},
+                "company_name": {"type": "string", "required": False},
+                "function": {"type": "string", "required": False},
+            }
+        )
+        return schema
+
+    def _get_base_search_domain(self):
+        return self._default_domain_for_partner_records(
+            # NOTE: here we must use `child_of` as default operator
+            partner_field="id",
+            operator="child_of",
+            with_backend=False,
```

Comment:
why `with_backend` false?

Reply:
This is the same as in the address service, we can't use `with_backend` here since the `expose_model` is `res.partner` and therefore has no `shopinvader_backend_id` attribute. The backend is now checked in `_get_invader_partner` anyway.